.github/workflows/master_post_message.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             -H "Accept: application/vnd.github+json" \
             -H "Authorization: Bearer $TOKEN" \
             -H "X-GitHub-Api-Version: 2022-11-28" \
-            "${{vars.GH_API_URL}}/repos/$REPO/issues/$PR_NUMBER/comments" | jq -r  '.[] | select(.body|test("^Commit hash.")) | .id')
+            "${{vars.GH_API_URL}}/repos/$REPO/issues/$PR_NUMBER/comments?per_page=100" | jq -r  '.[] | select(.body|test("^Commit hash.")) | .id')
```

Comment:
Loop not needed here, since this is the only comment posted in `$REPO/issues/$PR_NUMBER/comments` all other PR comments come under `$REPO/pulls/$PR_NUMBER/comments`